tests/unit/backend/corpora/dataset_processing/test_process.py

```diff
@@ -512,12 +533,19 @@ def mock_downloader_function(self, url, local_path, tracker, chunk_size):
             time.sleep(3)
 
     @patch("backend.corpora.dataset_processing.download.downloader")
-    @patch("backend.corpora.dataset_processing.process.get_file_info")
-    @patch("backend.corpora.dataset_processing.process.get_download_url_from_shared_link")
-    def test__dataset_tombstoned_while_uploading(self, mock_get_link, mock_get_size, mock_downloader):
+    @patch("backend.corpora.dataset_processing.process.from_url")
```

Comment:
Sorry, I can't quite understand this change? Could you explain it please?

Reply:
These tests weren't actually running because of a missing __init__ file. This test was failing because the function being tested changed and the functions being patched were no longer being called. This change just patches the main function to return an object with the properties needed to get the size and url.